lang/LangSource/PyrInterpreter3.cpp

```diff
@@ -1878,7 +1878,6 @@ HOT void Interpret(VMGlobals *g)
 				}
 				case 21 : {
 					-- sp ; // Drop
-					PyrSlot * vars = g->frame->vars;
 					SetRaw(&g->frame->vars[2], slotRawFloat(&g->frame->vars[2]) - 1.0); // dec i
```

Comment:
Or you can use vars in these next two lines, which would better match the code in other parts of the file.